app/views/updateExtractors.scala.html

```diff
@@ -264,11 +264,17 @@ <h1>Extractor Catalog</h1>
                                                 </li>
 
                                                 <!-- TODO: View Extractor Logs -->
+                                                @if(play.Play.application().configuration().getBoolean("viewextractorlogs")
+                                                        && play.Play.application().configuration().getString("clowder.log.serviceEndpoint")
+                                                        && play.Play.application().configuration().getString("clowder.log.username")
+                                                        && play.Play.application().configuration().getString("clowder.log.extractorNamePrefix")
```

Comment:
```suggestion
                                                @if(play.Play.application().configuration().getBoolean("viewextractorlogs", false)
                                                        && play.Play.application().configuration().getString("clowder.log.serviceEndpoint", "")
                                                        && play.Play.application().configuration().getString("clowder.log.username", "")
                                                        && play.Play.application().configuration().getString("clowder.log.extractorNamePrefix", "")
```

Reply:
why do we need the `viewextractorlogs` boolean. Will we ever define the other vars, and set vi viewextractorlogs to false?

Reply:
@bingzhang any thoughts regarding this?

Reply:
fixed